soap/src/java/com/zimbra/soap/admin/message/LicenseCacheServiceResponse.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * ***** BEGIN LICENSE BLOCK *****
+ * Zimbra Collaboration Suite Server
+ * Copyright (C) 2024 Synacor, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software Foundation,
+ * version 2 of the License.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License along with this program.
+ * If not, see <https://www.gnu.org/licenses/>.
+ * ***** END LICENSE BLOCK *****
+ */
+
+package com.zimbra.soap.admin.message;
+
+import com.zimbra.common.soap.AdminConstants;
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlAttribute;
+import javax.xml.bind.annotation.XmlRootElement;
+
+/**
+ * response class for LicenseCacheService
+ */
+@XmlAccessorType(XmlAccessType.NONE)
+@XmlRootElement(name=AdminConstants.E_LICENSE_CACHE_SERVICE_RESPONSE)
+public class LicenseCacheServiceResponse {
+
+    /**
+     * no-argument constructor wanted by JAXB
+     */
+    private LicenseCacheServiceResponse() {
+        new LicenseCacheServiceResponse((String) null);
+    }
+    public LicenseCacheServiceResponse(String isCacheRefreshed) {
+        this.isCacheRefreshed = isCacheRefreshed;
+
+    }
+
+    /**
+     * @zm-api-field-tag isCacheRefreshed
+     * @zm-api-field-description flag for is Cache Refreshed
+     */
+    @XmlAttribute(name=AdminConstants.IS_CACHE_REFRESHED /* isCacheRefreshed */, required=true)
+    String isCacheRefreshed = null;
```

Comment:
Can't we use declare this as a Boolean type.

Reply:
fixed